bluez-async/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Breaking changes
+
+- Added modalias to `AdapterInfo`.
```

Comment:
if you think there will be more fields to add later then you can make the struct nonexhaustive (I would need to look up how to do this though)

Reply:
I did consider that. The downside is it prevents users of the library from constructing a fake `AdapterInfo` for their tests.